### PR TITLE
ADD: Image input as filepath and base64

### LINKED ISSUE #41: Read image from filepath
Like Ruby / Python

### PR DESCRIPTION
**Pull request by sihrc**
Resolves #41

### CODE REVIEW

File: lib/image.js [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
+            );
+        } else if (typeof image === 'string') {
+            if (image.length <= 260 && fs.lstatSync(image).isFile()) {
+                input = image;
+                var arr = image.split('.');
+                options = arr[arr.length - 1]
+            } else {
+                input = new Buffer(image, 'base64');
+                options = 'png';
+            }
```
Comment (madisonmay) [inline lib/image.js]: What is the type of options?  It looks like it's either an object or a string.

File: lib/image.js [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+                  width: image.length
+                , height: image[0].length
+            };
+
+            console.warn(
+                "Image input as an array will be deprecated in the next major release"
+            );
+        } else if (typeof image === 'string') {
+            if (image.length <= 260 && fs.lstatSync(image).isFile()) {
+                input = image;
```
Comment (madisonmay) [inline lib/image.js]: Could we factor this out into a `fileType` function?

**Comment by madisonmay** [comment]
Few minor comments, then should be good to go!

File: lib/image.js (hunk 1 above)
Comment (sihrc) [inline lib/image.js]: It's either an object or a string

File: lib/image.js (hunk 2 above)
Comment (sihrc) [inline lib/image.js]: A function just returns if its a file?

**Comment by anneCarlson** [comment]
This loos good to me. I'm going to merge it in.

**Comment by sihrc** [comment]
thanks :+1:

**Pull request merged**